Pipfile

```diff
@@ -32,6 +32,9 @@ kerlescan = {editable = true,git = "https://github.com/beav/kerlescan.git",ref =
 bitmath = "*"
 jsonpatch = "*"
 insights-core = "*"
+nose = "*"
+mock = "*"
+coverage = "*"
```

Comment:
These should be pulled in via `packages-dev` above. Can you remove and see if things still work the same?

Reply:
You mean, `pipenv uninstall`? I couldn't run tests without adding those.

Reply:
the `packages` is used for things that are installed when the app is deployed, but `packages-dev` is for developer stuff like running tests. If you do `pipenv install --dev` it will pull in the developer dependencies. I'm guessing you did a `pipenv install nose` to get things running, but that pulls the test deps into the deployment.

To correct it, you can just edit out those three lines, run `pipenv lock` again to regen the lockfile, then do `pipenv install --dev` to install the developer deps

Reply:
Worked like a charm.